### PR TITLE
feat(t05): search job by city

### PR DESCRIPTION
**Pull request by HoangTrieu2803**

### CODE REVIEW

File: src/containers/Home/Jobs/ListJobs/index.tsx [hunk 1]
```diff
@@ -3,12 +3,14 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
...
 
 export default function ListJobs(props: { currentJobs: Job[] }) {
-    const { currentJobs} = props
+    const { currentJobs } = props
 
     return (
         <ul className='container-fluid mb-2' id='listJobs'>
             {currentJobs.map((job: Job, key: number) => {
                 const hours = Math.floor(Math.random() * 60) + 1
+                const city = ['Ho Chi Minh', 'Ha Noi', 'Da Nang']
```
[the remark this answers is not in the document]
Comment (HoangTrieu2803) [reply src/containers/Home/Jobs/ListJobs/index.tsx]: i've removed

File: src/containers/Home/Jobs/index.tsx [hunk 2]
```diff
@@ -37,7 +38,10 @@ export default function Jobs() {
 
     const searchJob = () => {
         return jobs.filter((searchJob) => {
-            return searchJob.jobName.toLocaleLowerCase().includes(search.toLocaleLowerCase())
+            return searchJob.jobName.toLocaleLowerCase().includes(search.title.toLocaleLowerCase())
+        }).filter((searchCity) => {
+            if (search.city === "All") return searchCity
+            else return searchCity.city.toLocaleLowerCase().includes(search.city.toLocaleLowerCase())
```
Comment (nqbinh13) [inline src/containers/Home/Jobs/index.tsx:44]: just `return` don't need else here

File: src/containers/Redux/search/searchSlice.ts [hunk 3]
```diff
@@ -1,10 +1,17 @@
 import { createAction, createReducer} from "@reduxjs/toolkit";
-export const searchAction = createAction <String> ("searchJob")
+export const searchAction = createAction <Search> ("searchJob")
+interface Search {
+    title: string;
+    city: string;
+}
```
Comment (nqbinh13) [inline src/containers/Redux/search/searchSlice.ts:6]: move this to type file

File: src/containers/Redux/search/searchSlice.ts [hunk 4]
```diff
@@ -1,10 +1,17 @@
...
 interface searchState {
-    search : String
+    search : Search
 }
```
Comment (nqbinh13) [inline src/containers/Redux/search/searchSlice.ts:9]: move this to type file

**Pull request merged**